### ISSUE TITLE
Identicon alternative ideas

### ISSUE THREAD

**Issue by skylarnagao** [PeerioTechnologies/peerio-client#237]
Goal is to improve usability and memorability of icons while maintaining entropy.

Consider other variables, e.g.

1) Icon in a color
2) Back ground in a different color
3) Background patterns (e.g. stripes, dots, waves, etc.)
4) Border (different color, different styles, e.g. one dotted and one solid black line)

e.g.
![secret-app-icon-set-vector](https://cloud.githubusercontent.com/assets/9948915/7192954/d5f59994-e468-11e4-88d8-5b7075735db5.png) but with backgrounds and borders.

**Comment by IpsmLorem** [comment]
:+1: 
Another approach, could be investigated too.
Why not use the current cryptoaptterne as a deformation layer for a user chosen contact picture ?
The avatar would be: the user picture + crypto layer = right displayed user chosen pic.
If the crypto layer change, the result picture is no more the chosen pic.

**Comment by skylarnagao** [comment]
The thing is, avatars really do need to be a constant for users. 

So is users uploaded their own images, how could they change their avatars with this approach? Like, say I decide I don't want a photo of myself, but want an icon. When this changes, it creates the impression that my account or the connection has been compromised somehow.

I think if we go with user-chosen avatars, they should simply be in conjunction with the identicons. So, you see the standard avatar, hover your cursor over it and then see the identicon. This has the benefit of maintaining the identicon, allowing users to upload their own avatars, and change them, but the consequence of pushing identicons out of view and less likely to be memorized by users. 

I suppose, some users don't care as much, and it could be optional whether your identicon or your avatar is displayed by default.

**Comment by IpsmLorem** [comment]
No, when I introduced a user chosen avatar, I was only refering to the local account (the recipients) et not the sender.
A peerio user may choose a contact avatar that suits their need.
